ecommerce-demo/workspaces/childDemo_clothing/workspace.json

```diff
@@ -0,0 +1,17 @@
+{
+  "data": {
+    "attributes": {
+      "name": "3. Demo Workspace / Customer B (Child)"
+    },
+    "id": "ecommerce-merchant-clothing",
```

Comment:
Should we add `demo_` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_ecommerce_merchant_clothing",
```